### ISSUE TITLE
Missing corners in connected textures

### ISSUE THREAD

**Issue by Ezoteric** [halvors/Quantum#19]
Hey, its cool you have these CTs but your missing quite a few.

For example the UL needs a small corner to connect when shaped as a L and it needs a lot more cause the same UL will also need one without a dot to connect to a 2 layer L.

![2018-04-12_15 54 34](https://user-images.githubusercontent.com/14137626/38681881-117f4506-3e6a-11e8-9224-69114a4fe1fa.png)

On the Pic you can see what i mean.
In the red circle you can see its missing its connector, so when i fixed that by adding it like in the blue circle but that creates the following error you can see in the green circle cause it should be like the pink circle.

So you kinda miss a whole set of the green ones ;)

PS: Dont look at the other mistakes on the glass border textures as i m still sphaxing it correctly :P

**Comment by halvors** [comment]
@Ezoteric Thanks for your bug report!

The corners was never added due to how connected textures work. But that could be fixed in the future.
As far as i can understand that was the only missing piece? (But in both electromagnets and gas funnel of course).

**Comment by Ezoteric** [comment]
well it for all connected textures. And you also miss corners on just glass with a corner :P 

If i got some time ill look for a mod that does have ALL CTMs so you know which you actually mis cause there is quite a lot of them  :P

**Comment by halvors** [comment]
With edged glass? The nature of how conncted textures is implemented is the problem. There is only a way of telling when theres a neighbor block in over, under, north, east, south, west. 

So it would be difficult to get rid of the "center" dot as there is no way of detecting the last block of the square.

**Comment by Ezoteric** [comment]
Well its not a major problem it just looks stupid :P 
But it should be possible as there are mods with glass and they dont have missing corners.
I also looked at a few mods but seems they all have different approaches to fix that.

So i dunno, you can keep this here till you finished the important stuff first :)

**Comment by halvors** [comment]
Yeah, think the ConnectedTexturesMod does some ASM injection of minecraft classes to do this.
@Ezoteric Do you know of any texture packs that adds connected texture glass and also works on vanilla Minecraft?

**Comment by Ezoteric** [comment]
Sphax for vanilla MC has 47 glass textures for the normal glass block :P 
In the textpack they are located : assets\minecraft\mcpatcher\ctm\glass\frame\

BTW chisel only has 1 file with 4 of them on it and Extrautils has 1 file with 5 of them on it.
Kinda weird :P
